source/packages/freeradius-package.rst

```diff
@@ -0,0 +1,152 @@
+- Auth with PEAP + dynamic VLAN assignment
+- Auth with EAP-TLS/EAP-TTLS + dynamic VLAN assignment
+
+  .. code::
+
+    radiusd[3206]: Login OK: [testuser/<via Auth-Type = EAP>] (from client pfsense port 0 cli 00-04-23-5C-9D-19)
+    radiusd[3206]: Login OK: [testuser/<via Auth-Type = EAP>] (from client pfsense port 0 cli 00-04-23-5C-9D-19)
+    radiusd[3206]: Login OK: [testuser/<via Auth-Type = EAP>] (from client pfsense port 0 via TLS tunnel)
+    radiusd[3206]: Login OK: [testuser/<via Auth-Type = EAP>] (from client pfsense port 0 via TLS tunnel)
+
+- Simultaneous-Use - The following will be present in the system log::
+
+    radiusd[3206]: Multiple logins (max 1) : [testuser/testpw] (from client testing port 10)
+
+- A certain amount of time per day/week/month/forever (CHECK-ITEM:
+  Max-Daily-Session := 60 ) The user will be disconnected and cannot
+  re-login after the amount of time is reached::
+
+    radiusd[3206]: Invalid user (rlm_counter: Maximum daily usage time reached): [testuser/<via Auth-Type = EAP>] (from client pfsense port 0 cli 00-04-23-5C-9D-19)
+
+- A certain amount of traffic per day/week/month/forever. The user will
+  be disconnected and cannot re-login after the amount of traffic is
+  reached. The syslog output looks like this::
+
+    root: FreeRADIUS: Used amount of daily upload and download traffic by testuser is 0 of 100 MB! The user was accepted!!!
+    root: FreeRADIUS: Credentials are probably correct but the user testuser has reached the daily amount of upload and download traffic which is 243 of 100 MB! The user was rejected!!!
+
+- MySQL
+- LDAP/ActiveDirectory (connecting to MS AD with PAP)
+- User-Auth with SQUID
+- One-Time-Password
+
+
+FreeRADIUS General Configuration
+--------------------------------
+
+Related Articles
+^^^^^^^^^^^^^^^^
+
+Refer to the following articles for more information on the listed topics:
+
+-  :doc:`Testing FreeRADIUS </packages/testing-freeradius>`
+-  :doc:`Mobile One-time Passwords with FreeRADIUS </packages/mobile-one-time-passwords-with-freeradius>`
+-  :doc:`Using Captive Portal with FreeRADIUS </captiveportal/captive-portal-configuration>`
+-  :doc:`Using EAP and PEAP with FreeRADIUS </packages/using-eap-and-peap-with-freeradius>`
+-  :doc:`Using OpenVPN With FreeRADIUS </vpn/openvpn/authenticating-openvpn-users-with-freeradius>`
+-  :doc:`Using Squid with FreeRADIUS </cache-proxy/using-squid-with-freeradius>`
+
+Get FreeRADIUS Status Server Updates
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The status server will give lots of information about the FreeRADIUS
+server. Many stats are shown about Accounting-Packets, dropped packets
+and much more. To enable status server and request information from the
+server do the following:
+
+- Setup an interface with **Interface-Type**: *status* and a free port.
+  I choose **Port**: ``1818`` in this how-to.
+- Setup a NAS/Client with **IP-Address**: ``127.0.0.1`` and a password.
+  I choose *testing123* in this how-to.
```

Comment:
Change `I choose` here again, see my previous note.

Reply:
I'll update this. However I didn't make change to this file (I simply renamed `usermanager/freeradius-2-x-package.rst` to `source/packages/freeradius-package.rst`).